tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -863,6 +863,25 @@ private function skipOci8AndMysqli()
             $this->markTestSkipped('Mysqli driver dont support this feature.');
         }
     }
+
+    public function testIteratorCanBeIteratedMultipleTimes()
+    {
+        $sql = "SELECT test_int, test_string FROM fetch_table";
+        $stmt = $this->_conn->query($sql);
+        $stmt->setFetchMode(\PDO::FETCH_ASSOC);
+
+        $i = 0;
+        foreach ($stmt as $row) {
+            $i++;
+        }
+
+        $j = 0;
+        foreach ($stmt as $row2) {
+            $j++;
+        }
+
+        $this->assertEquals($i, $j);
```

Comment:
I think this is likely because of the MySQL version. 5.6 and up seems to support it. Travis is on a pre 5.6 version of MySQL IIRC although I think the new trusty boxes may be newer. So I guess we could update travis to build against the newer versions if we want (if this is indeed the issue!).